src/main/java/com/snowflake/kafka/connector/internal/streaming/DirectStreamingClientHandler.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2023 Snowflake Inc. All rights reserved.
```

Comment:
nit: 2024

Reply:
It was just a file rename. So, I blame Github here for being unable to figure it out.